.gitignore

```diff
@@ -6,4 +6,5 @@
 atlassian-ide-plugin.xml
 com_crashlytics_export_strings.xml
 crashlytics.properties
-crashlytics-build.properties
\ No newline at end of file
+crashlytics-build.properties
+typings
```

Comment:
added typings because 'node' typings were missing -> I was forced to install them locally